CHANGELOG.md

```diff
@@ -2,6 +2,120 @@
+Nexus repo setup script missing in ods-core ([#423](https://github.com/opendevstack/ods-core/issues/423))
+ods-core/infrastructure-setup/ still contains rundeck setup ([#424](https://github.com/opendevstack/ods-core/issues/424))
+Not all params from configuration-sample are used anymore ([#376](https://github.com/opendevstack/ods-core/issues/376))
+Webhook Proxy does not run under jenkins serviceaccount ([#413](https://github.com/opendevstack/ods-core/issues/413))
+Jenkins runs out of memory and becomes unresponsive ([#412](https://github.com/opendevstack/ods-core/issues/412))
+CNES report plugin in base slave incompatible with ods-core SQ version ([#419](https://github.com/opendevstack/ods-core/issues/419))
+checkout-respositories.sh executes in sh instead of bash ([#397](https://github.com/opendevstack/ods-core/issues/397))
+webhook proxy - jenkins fails to fetch build in case branch name > 63chars ([#369](https://github.com/opendevstack/ods-core/issues/369))
+Download of sonar-scanner-cli fails ([#378](https://github.com/opendevstack/ods-core/issues/378))
+fix volume size and fix unresolved mirror errmsg on startup ([#639](https://github.com/opendevstack/ods-core/pull/639))
+Sonarqube 8.2 support - missing jacoco plugin ([#457](https://github.com/opendevstack/ods-core/issues/457))
+Use latest version (1.0.44) of openshift-sync plugin ([#461](https://github.com/opendevstack/ods-core/pull/461))
+Do not copy init.groovy.d files on initial boot ([#443](https://github.com/opendevstack/ods-core/issues/443))
+Restrict permissions of the default role ([#452](https://github.com/opendevstack/ods-core/issues/452))
+Update Nexus to 3.20.0 ([#346](https://github.com/opendevstack/ods-core/issues/346))
+Better documentation, logging and env var handling ([#446](https://github.com/opendevstack/ods-core/pull/446))
+Move ODS config to Jenkins master ([#410](https://github.com/opendevstack/ods-core/pull/410))
+Ensure that pipeline name is not longer than 63 characters ([#405](https://github.com/opendevstack/ods-core/pull/405))
+Add missing PROV_APP_CROWD_URI param ([#384](https://github.com/opendevstack/ods-core/pull/384))
+Add missing SONAR_ADMIN_PASSWORD_B64 param ([#382](https://github.com/opendevstack/ods-core/pull/382))
+Use HTTPS in curl command ([#387](https://github.com/opendevstack/ods-core/pull/387))
+
+### Removed
+remove inactive spring active profile property from prov app config map ([#64](https://github.com/opendevstack/ods-core/issues/648))
+Remove image trigger from Jenkins master instances ([#210](https://github.com/opendevstack/ods-core/issues/210))
+Remove obsolete scripts from infrastructure-setup ([#618](https://github.com/opendevstack/ods-core/pull/618))
+removed owasp-dependency-check pvc and cli ([#556](https://github.com/opendevstack/ods-core/pull/556))
+Remove image trigger from Jenkins deployments ([#396](https://github.com/opendevstack/ods-core/issues/396))
```

Comment:
Same as 210 above, so can be merged into one, and then I'd put it into "changed"